src/components/credit-card-number/code/templates/creditCardNumberTemplate.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+
+module.exports = function(component) {
+
+  var label, visa, mastercard, amex, discover, images;
+
+  if (component.props.label){
+    label = ( <label className="input-group-label credit-card-number-label" htmlFor={component.props.id}>{component.props.label}</label> );
+  }
+
+  if(component.props.showImages)
+  {
```

Comment:
Keep the { on the same lie as the if, as you have done in the previous if statement. Same applies below.